src/components/Actions.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable linebreak-style */
+function getUserCountryCodeFromIp() {
+  let userCountryCode = 'CHN';
+  const url = 'https://get.geojs.io/v1/ip/country.json';
+  return fetch(url)
+    .then((res) => res.json())
+    .then((data) => {
+      userCountryCode = data.country;
+      return userCountryCode;
+    });
+}
```

Comment:
this might be not that important, but the `actions` aren't really a component. Either you move it to an `actions` folder or `utils`, `effects`, `hooks` whatever the name it is. It has not to be misleading to what this file does